src/index.js

```diff
@@ -0,0 +1,162 @@
+    
+    updateDisplay()
+}; 
+
+const reduceOneTemp = () => {
+    temperature -= 1;
+    tempDisplay.innerText = temperature;
+
+    updateDisplay()
+};
+
+increaseTemp.addEventListener('click', addOneTemp);
+decreaseTemp.addEventListener('click', reduceOneTemp);
+
+const updateCityName = () => {
+    const cityText = cityNameInput.value;
+    cityNameDisplay.innerText = cityText;
+};
+
+cityNameInput.addEventListener('input', updateCityName);
+
+
+// function that gets weather data
+const getWeather = (lat, lon) => {
+    return axios
+        .get(`http://localhost:5000/weather?lat=${lat}&lon=${lon}`)
+      .then((response) => {
+            return response.data;
+        })
+        .catch((error) => {
+            console.log(`Error is: ${error}`);
+        });
+};
+
+// function that gets location 
+const getLocation = (cityName) => {
+    return axios
+        .get(`http://localhost:5000/location?q=${cityName}`)
+        .then((response) => {
+          return response.data[0];
+        })
+        .catch((error) => {
+            console.log(`Error is: ${error}`);
+        });
+};
+
+// function that updates current temp using location and weather data
+const getCurrentTemp = async () => {
+    try {
+        const cityName = cityNameDisplay.innerText;
+        const locationData = await getLocation(cityName);
+        const weatherData = await getWeather(locationData.lat, locationData.lon);
+        const temperatureKelvin = weatherData.main.temp;
+        temperature = Math.round((temperatureKelvin - 273.15) * 9/5 + 32); 
+        tempDisplay.innerText = temperature;
+        updateDisplay();
+    }
+    catch(error) {
+        console.log(error);
+    };
+};
+tempButton.addEventListener('click', getCurrentTemp);
+
+// sky options with corresponding clouds
+const skyOptions = [
+    { name: 'Sunny', display: '☁️ ☁️ ☁️ ☀️ ☁️ ☁️' },
+    { name: 'Cloudy', display: '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️' },
+    { name: 'Rainy', display: '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧' },
+    { name: 'Snowy', display: '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨' }
+];
+
+// create option element in skySelect area
+skyOptions.forEach(option => {
+    const optionElement = document.createElement('option');
+    optionElement.value = option.name.toLowerCase();
+    optionElement.textContent = option.name;
+    skySelect.appendChild(optionElement);
+});
+
+const updateSky = (event) => {
+    // remove all possible sky classes
+    gardenContent.classList.remove('sunny', 'cloudy', 'rainy', 'snowy');
+    // add the selected sky class to gardenContent so that background color also changes
+    gardenContent.classList.add(event.target.value);
+    // update sky display with emojis
+    const selectedOption = skyOptions.find(option => option.name.toLowerCase() === event.target.value);
+    skyDisplay.textContent = selectedOption.display;
+};
+
+skySelect.addEventListener('change', updateSky);
+// start with sunny
+gardenContent.classList.add('sunny');
+skyDisplay.textContent = skyOptions[0].display;
```

Comment:
Prefer this logic to be put in a helper function that's descriptively named. TBH, if your comment wasn't on line 148 I wouldn't know what the intent of lines 149-150 is. If code cannot be understood without a comment, it should be refactored so it can be understood without a comment. 

```suggestion
const setInitialSky = () => {
  gardenContent.classList.add('sunny');
  skyDisplay.textContent = skyOptions[0].display;
};
```